open-metadata-implementation/access-services/data-engine/data-engine-api/src/main/java/org/odpi/openmetadata/accessservices/dataengine/model/TabularColumn.java

```diff
@@ -0,0 +1,215 @@
+
+/**
+ * OM type TabularColumn
+ */
+@JsonAutoDetect(getterVisibility = PUBLIC_ONLY, setterVisibility = PUBLIC_ONLY, fieldVisibility = NONE)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TabularColumn implements Serializable {
+
+    //Referenceable
+    protected String qualifiedName;
+    protected Map<String,String> additionalProperties;
+
+    //SchemaElement
+    protected String anchorGuid;
+    protected String displayName;
+    protected String description;
+
+    //SchemaAttribute
+    protected int position;
+    protected int minCardinality;
+    protected int maxCardinality;
+    protected boolean allowsDuplicateValues;
+    protected boolean orderedValues;
+    protected String defaultValueOverride;
+    protected String nativeClass;
+    protected String name;
+    protected String[] aliases;
+    protected DataItemSortOrder dataItemSortOrder;
+
+    public String getQualifiedName() {
+        return qualifiedName;
+    }
+
+    public void setQualifiedName(String qualifiedName) {
+        this.qualifiedName = qualifiedName;
+    }
+
+    public Map<String, String> getAdditionalProperties() {
+        return additionalProperties;
+    }
+
+    public void setAdditionalProperties(Map<String, String> additionalProperties) {
+        this.additionalProperties = additionalProperties;
+    }
+
+    public String getAnchorGuid() {
+        return anchorGuid;
+    }
+
+    public void setAnchorGuid(String anchorGuid) {
+        this.anchorGuid = anchorGuid;
+    }
+
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    public void setDisplayName(String displayName) {
+        this.displayName = displayName;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void setPosition(int position) {
+        this.position = position;
+    }
+
+    public int getMinCardinality() {
+        return minCardinality;
+    }
+
+    public void setMinCardinality(int minCardinality) {
+        this.minCardinality = minCardinality;
+    }
+
+    public int getMaxCardinality() {
+        return maxCardinality;
+    }
+
+    public void setMaxCardinality(int maxCardinality) {
+        this.maxCardinality = maxCardinality;
+    }
+
+    public boolean isAllowsDuplicateValues() {
+        return allowsDuplicateValues;
+    }
+
+    public void setAllowsDuplicateValues(boolean allowsDuplicateValues) {
+        this.allowsDuplicateValues = allowsDuplicateValues;
+    }
+
+    public boolean isOrderedValues() {
+        return orderedValues;
+    }
+
+    public void setOrderedValues(boolean orderedValues) {
+        this.orderedValues = orderedValues;
+    }
+
+    public String getDefaultValueOverride() {
```

Comment:
I am not sure what these field are - can you add javadoc. For example I am not sure what default value override means.

Reply:
No longer applies, type doesn't exist